### DISCUSSION TITLE
Cannot copy and paste in copyq

### DISCUSSION THREAD

**Discussion by ghost** [hluk/CopyQ#2077] [Q&A]
I don't know how I can set shortcuts to `copy and paste` in my copy q. I already went to `File > Preferences > Shortcuts` but still I couldn't figure out how to do that. I read the whole wiki regarding copyq but still I don't even have a clue how can I set `Ctrl+c` for copy and `Ctrl+v` for paste. Options in the shotcuts tab looks confusing and I don't know how to set my shortcut. Any help will be appreciated.

**Comment by laur89** [comment]
Don't think you should be explicitly setting these shortcuts to begin with - this should be particular application or system-wide setting.

1. does copy-pasting text within say your browser work? (ignore copyq at this point)
2. have you tried reverting copyq configuration to default to make sure you haven't messed something up?
3. what OS are you on? 
4. what version of copyq are you running? (run `copyq --version`)
5. what actually happens when you copy text whilst copyq is running? Is copyq list simply empty?
6. if you run `copyq add test-text` in terminal, can you see it appear in copyq then?
